tempesta_fw/http.h

```diff
@@ -234,6 +234,7 @@ typedef struct {
 #define TFW_HTTP_HAS_HDR_DATE		0x020000	/* Has Date: header */
 /* It is stale, but pass with a warning */
 #define TFW_HTTP_RESP_STALE		0x040000
+#define TFW_HTTP_MSG_SENT		0x080000
```

Comment:
This is the section for response flags. I believe that it should be moved to section for request flags (and renamed to `TFW_HTTP_REQ_SENT).